src/pas/plugins/ldap/cache.rst

```diff
@@ -8,7 +8,8 @@ Setup
 Basics
 ------
 
-::
+.. code-block:: pycon
```

Comment:
Shouldn't `pycon` be `python`? Hm, it works though when I try it with `restview`, and when I use a random string instead it gives an error. So `pycon` is apparently okay.
I see very minor differences. Is `pycon` maybe meant as variant that shows better in presentations, for example on the PyCon conference? Do you know that? Just curious. :-)

Reply:
pycon is python console

Reply:
Ah! Thanks for the info.